pcdsdevices/lamp_motion.py

```diff
@@ -0,0 +1,42 @@
+"""
+LAMP Motion Classes
+
+This module contains classes related to the TMO-LAMP Motion System
+"""
+
+from ophyd import Component as Cpt
+from ophyd import Device
+
+from .epics_motor import BeckhoffAxis
+from .interface import BaseInterface
+
+
+class LAMP(BaseInterface, Device):
+    """
+    LAMP Motion Class
+
+    This class controls motors fixed to the LAMP Motion system
+
+    Parameters
+    ----------
+    prefix : str
+        Base PV for the LAMP motion system
+
+    name : str
+        Alias for the device
+    """
+    # UI representation
+    _icon = 'fa.minus-square'
```

Comment:
Add `tab_component_names = True` in the class body to lessen tab completion items in the IPython session

Reply:
Resolved in commit 16ee871